JapaneseFilter/sections/specific.txt

```diff
@@ -45,6 +45,7 @@ finders.me##.__ad-pc
 finders.me##div[class^="__super-banner-"]
 soco-st.com##.ad_01
 soco-st.com##.ad_02
+ticketjam.jp##.l-section
```

Comment:
Do not block this, it just happened to be empty on the posted page but shows useful tags on other pages such as `https://ticketjam.jp/magazine/music/singer-song-writer/15520`.